### DISCUSSION TITLE
Let end users read Azure Analysis Services model metadata

### DISCUSSION THREAD

**Discussion by bdarbonneau** [DaxStudio/DaxStudio#1132] [Q&A] [answered]
When using DAX studio to build DAX queries against Azure Analysis Services models, it is helps to be able to read the measure definitions by way of the 'Define measure' feature. But for end users, it returns nothing.
Is there a way - except giving admin role - to grant end users permission to read an Azure Analysis Services model metadata ?

**Comment by dgosbell** [comment, marked as answer]
No, the only people that can read measure definitions are people with write access to the model (ie. people with admin rights). This is a Microsoft restriction we cannot bypass it from the DAX Studio side.
